From 9be293abd94b014bce04b0384cf903832f01fad3 Mon Sep 17 00:00:00 2001
From: CI Assistant <yossi.ilkanaev@intel.com>
Date: Thu, 19 Jul 2018 08:48:50 +0200
Subject: [PATCH] Merge pull request #76 in SW_UGW/linux from
 bugfix/DRVLIB_SW-817-pause-frame-sent-out-only-few-packet-are-recieved to
 master

* commit 'f39c89831c665eb19eb07cf0d669571846428c9b':
  DRVLIB_SW-817:Disable IP OCC feature
---
 drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
index 0c60b28d0578..1dc37526dab0 100644
--- a/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
+++ b/drivers/net/ethernet/lantiq/cqm/falconmx/cqm.c
@@ -3688,6 +3688,7 @@ static int cqm_falconmx_probe(struct platform_device *pdev)
 	if (cbm_hw_init(pdev))
 		return -1;
 	configure_ports(cqm_ctrl->cqm_cfg);
+	cbm_w32(cqm_ctrl->enq + IP_OCC_EN, 0);
 	cbm_w32((cqm_ctrl->cqm + CBM_BSL_CTRL), CBM_BSL_CTRL_BSL1_EN_MASK |
 		CBM_BSL_CTRL_BSL2_EN_MASK);
 	/*Enable the EQM and DQM contollers*/
